libs/geospatial/src/main/java/org/codice/ddf/libs/geo/util/GeospatialUtil.java

```diff
@@ -263,4 +274,66 @@ public static Geometry transformToEPSG4326LonLatFormat(
     }
     return transformedGeometry;
   }
+
+  /**
+   * Create a circular polygon from a lat, lon, radius, vertices, and distanceTolerance.
+   *
+   * @param lat - latitude in EPSG:4326 decimal degrees
+   * @param lon - longitude in EPSG:4326 decimal degrees
+   * @return radius - distance from the center point in meters
+   * @return maxVertices - the maximum number of vertices in the finished circle polygon
+   * @return distanceTolerance - the maximum distance from the original vertices a reduced vertex
```

Comment:
✏️ 	I would include something like this in the description here https://github.com/codice/ddf/pull/4037/files#diff-37861fa17bc91ac8731cb8f876732f1aR60

Reply:
remove duplicate @return tags?